samples/rxfilter/rxfilter.c

```diff
@@ -4,7 +4,7 @@
 //
 
 #include <xdpapi.h>
-
+#include <ws2tcpip.h>
```

Comment:
there's something of a tradition to try to `#include` in alphabetical order - if it works in that order, please rearrange it.

with Windows headers, though, it's often trickier than that.